### ISSUE TITLE
bug: pre-commit args not passed correctly

### ISSUE THREAD

**Issue by ryanmerolle** [aristanetworks/j2lint#219]
```yaml
---
repos:
  - repo: https://github.com/aristanetworks/j2lint
    rev: v1.2.0
    hooks:
      - id: j2lint
        args: ["-i", "jinja-statements-delimiter"]
```

```bash
Check for Linting error on Jinja2 templates..............................Failed
- hook id: j2lint
- exit code: 2

  usage: j2lint [-h] [-l] [-r RULES_DIR] [-v] [-e EXTENSIONS] [-d] [-j] [-s]
                [--log] [--version] [-o]
                [-i [{jinja-syntax-error,single-space-decorator,operator-enclosed-by-spaces,jinja-statements-single-space,jinja-statements-indentation,jinja-statements-no-tabs,single-statement-per-line,jinja-statements-delimiter,jinja-variable-lower-case,jinja-variable-format,S0,S1,S2,S3,S4,S5,S6,S7,V1,V2} ...]]
                [-w [{jinja-syntax-error,single-space-decorator,operator-enclosed-by-spaces,jinja-statements-single-space,jinja-statements-indentation,jinja-statements-no-tabs,single-statement-per-line,jinja-statements-delimiter,jinja-variable-lower-case,jinja-variable-format,S0,S1,S2,S3,S4,S5,S6,S7,V1,V2} ...]]
                [FILE ...]
  j2lint: error: argument -i/--ignore: invalid choice: 'templates/configs/edges/cisco_ios.j2' (choose from jinja-syntax-error, single-space-decorator, operator-enclosed-by-spaces, jinja-statements-single-space, jinja-statements-indentation, jinja-statements-no-tabs, single-statement-per-line, jinja-statements-delimiter, jinja-variable-lower-case, jinja-variable-format, S0, S1, S2, S3, S4, S5, S6, S7, V1, V2)
```

**Comment by gmuloc** [comment]
Hey - thanks for reporting this is documented here: https://github.com/aristanetworks/j2lint#using-j2lint-as-a-pre-commit-hook

You should use:

```
---
repos:
  - repo: https://github.com/aristanetworks/j2lint
    rev: v1.2.0
    hooks:
      - id: j2lint
        args: ["-i", "jinja-statements-delimiter", "--"]
```

this is because the way argparse is configured, the filename is seen as an argument to `-i`
